src/content/texts.json

```diff
@@ -41,6 +45,8 @@
 
       "anonymousDonor": {
         "name": "Anonym givare",
+        "companyName": "Anonym givare",
+        "organizationNumber": "",
```

Comment:
Not sure what would make sense to have here, or if companies should even be allowed to be anonymous

Reply:
Does seem like quite a strange state that perhaps we shouldn't allow